### ISSUE TITLE
Age and Geological_Time relation

### ISSUE THREAD

**Issue by alexandretessarollo** [own-pt/own-en#5]
I’m finishing the geological time terms (based on [ICSchart](http://www.stratigraphy.org/ICSchart/ChronostratChart2020-01.jpg)). Here’s my model:
Aptian Age is Instance Hypernym of Age
Aptian Age is partHolonymOf Lower_Cretaceous
Age is partHolonymOf Epoch

Question: when it comes to the relation between Age and Geological_Time, which one should it be? I believe it should be a hypernym, but I’m couldn’t rule out Member of this domain - TOPIC completely. I don’t think it should be instance-hyper (age is not as unique as Aptian or Barack Obama) or partHolonymOf (age and epoch are not part of geological time, but more like types of geological time, hence why I’m prone to hypernym)

**Comment by arademaker** [comment]
geological_time is http://wn.mybluemix.net/synset?id=15116283-n. It has  epoch, geological_period, geological_era, and aeon as hyponyms. This is already there, right?  It was similar to the ISC model where these are the types and  particular periods (in the general sense) like Aptian are instances of these types. Moreover, these types are also connectecd by `Part holonym` and `Part meronym`. 

I think we should not introduce fine grain domain classification. Do you agree with making all these synsets (related to the geological times) member of the domain http://wn.mybluemix.net/synset?id=06115701-n?

Are you following http://arademaker.github.io/bibliography/ontobras-2019-sumo.html and http://arademaker.github.io/bibliography/gwc-2018-chronostratigraphic.html? Not sure what are you questioning regarding these previous work. It is fine if you find some mistake in the paper, but is that the case?

For each period X (instance of some type), we need to establish (and I believe the articles have already done that) the possible relations:

1. being a part of another period (in the chart, the left box)
2. being instance of a particular columnn (in the chart: eon, period etc)
3. do we have any relation to connect a period with the previous and  next one in the same column of the chart?

**Comment by alexandretessarollo** [comment]
> geological_time is http://wn.mybluemix.net/synset?id=15116283-n. It has epoch, geological_period, geological_era, and aeon as hyponyms. This is already there, right? It was similar to the ISC model where these are the types and particular periods (in the general sense) like Aptian are instances of these types. Moreover, these types are also connectecd by `Part holonym` and `Part meronym`.

Yes, age and geological_time are connected by a hypernym relation; Aptian is an instance of age; age and epoch are holonym part; Aptian and Lower_Cretaceous are holonym part.

My question was just about the age / geological_time relation, if it really should be hypernym.  I couldn’t rule out Member of this domain - TOPIC.

Is it possible/reasonable to connect age and geological_time with both hypernym AND Member of this domain - TOPIC?

> I think we should not introduce fine grain domain classification. Do you agree with making all these synsets (related to the geological times) member of the domain http://wn.mybluemix.net/synset?id=06115701-n?

Counterproposal: why not keep GBA rocks be under lithology and then put geological_time and lithology under geology? (all domain - TOPIC relations). This way WN would be better organized and with all of these synsets connected in a hierarchical fashion. This way we wouldn't put limestone and Aptian in the same "geology bag", but they would both be under geology somehow. I think it might even help WSD (provided they are able to run the entire graph of relations, not just X steps).


> Are you following http://arademaker.github.io/bibliography/ontobras-2019-sumo.html and http://arademaker.github.io/bibliography/gwc-2018-chronostratigraphic.html? Not sure what are you questioning regarding these previous work. It is fine if you find some mistake in the paper, but is that the case?

I used ISC 2020/1. It has the same data used in our SUMO extension, except for the Chibanian. A changelog for the different ISC versions can be found at [http://www.stratigraphy.org/ICSchart/ChangeLog2012-2013-2014-2015-2016-2017-2018-2019-2020.txt](http://www.stratigraphy.org/ICSchart/ChangeLog2012-2013-2014-2015-2016-2017-2018-2019-2020.txt). SUMO was extende with 2018/07.

So far I've found no mistakes in the works you mentioned.

> For each period X (instance of some type), we need to establish (and I believe the articles have already done that) the possible relations:
> 
> 1. being a part of another period (in the chart, the left box)
> 2. being instance of a particular columnn (in the chart: eon, period etc)
> 3. do we have any relation to connect a period with the previous and  next one in the same column of the chart?

1 and 2 already ok and implemented. About 3 I don't know if WN has any relation we could use to say "Jurassic is right after Triassic" and/or "Jurassic is right before Cretaceous". It would be of great value, though. I'm gonna go through PWN definitions again, ideas and suggestions are welcome.

**Comment by arademaker** [comment]
just to document here our conversation:

1. we need to better understand the domains in WN
2. I agree that after (1) we can use lithology as domain, if (1) shows us that this make sense.
3. it would be interesting to move forward with https://github.com/CGI-IUGS/timescale-data/pull/2. that could potentially help with the discussion about future maintenance of our work in the face of updates in the GBA and ISC.
4. as far as I know, we don't have any temporal relation in PWN. I believe this is what we concluded as motivation for the inclusion of these terms in SUMO. We don't have relations between the decades (see http://wn.mybluemix.net/synset?id=15204983-n).
